yt/geometry/tests/test_particle_deposit.py

```diff
@@ -34,6 +34,7 @@ def test_one_zone_octree_deposit():
     assert sp["deposit", "io_cic"].shape == (1,)
 
 
+@requires_module("h5py")
```

Comment:
Good catch.  I think we'd relied on some (not implemented!) transitive dependence in the isogal.